src/OrleansServiceFabricUtils/OrleansCommunicationListener.cs

```diff
@@ -42,7 +42,8 @@ public class OrleansCommunicationListener : ICommunicationListener
         /// </summary>
         /// <param name="context">The context.</param>
         /// <param name="configuration">The configuration.</param>
-        public OrleansCommunicationListener(ServiceContext context, ClusterConfiguration configuration)
+        /// <param name="siloName">The silo name, or <see langword="null"/> to have a name automatically selected.</param>
+        public OrleansCommunicationListener(ServiceContext context, ClusterConfiguration configuration, string siloName = null)
```

Comment:
I see , yeah I meant ClusterConfiguration.Overrides.[...].SiloName. 
I was just confused since there seems to be two source of truth for siloName.

but the rest LGTM.